tools/accuracy_checker/accuracy_checker/adapters/pose_estimation_3d.py

```diff
@@ -0,0 +1,243 @@
+            persons_keypoints_y.append(keypoints_y)
+            persons_keypoints_v.append(keypoints_v)
+
+        persons_keypoints_x = np.array(persons_keypoints_x)
+        persons_keypoints_y = np.array(persons_keypoints_y)
+        persons_keypoints_v = np.array(persons_keypoints_v)
+        scores = np.array(scores)
+
+        return persons_keypoints_x, persons_keypoints_y, persons_keypoints_v, scores
+
+    @staticmethod
+    def _get_root_relative_poses(features, found_poses_2d):
+        poses_2d = []
+        num_kpt_panoptic = 19
+        num_kpt = 18
+        for pose_id in range(found_poses_2d.visibility.shape[0]):
+            if found_poses_2d.visibility[pose_id, 1] == 0:  # skip pose if does not found neck
+                continue
+            # just repacking
+            pose_2d = np.ones(num_kpt_panoptic * 3 + 1, dtype=np.float32) * -1  # +1 for pose confidence
+            for kpt_id in range(num_kpt):
+                if found_poses_2d.visibility[pose_id, kpt_id] != 0:
+                    pose_2d[map_id_to_panoptic[kpt_id] * 3] = found_poses_2d.x_values[pose_id, kpt_id].astype(np.int32)
+                    pose_2d[map_id_to_panoptic[kpt_id] * 3 + 1] = found_poses_2d.y_values[pose_id, kpt_id].astype(np.int32)
+                    pose_2d[map_id_to_panoptic[kpt_id] * 3 + 2] = found_poses_2d.visibility[pose_id, kpt_id]
+            pose_2d[-1] = found_poses_2d.scores[pose_id]
+            poses_2d.append(pose_2d)
+
+        keypoint_treshold = 0.1
+        poses_3d = np.ones((len(poses_2d), num_kpt_panoptic * 4), dtype=np.float32) * -1
+        for pose_id in range(len(poses_3d)):
+            if poses_2d[pose_id][2] > keypoint_treshold:
+                neck_2d = poses_2d[pose_id][:2].astype(int)
+                # read all pose coordinates at neck location
+                for kpt_id in range(num_kpt_panoptic):
+                    map_3d = features[kpt_id * 3:(kpt_id + 1) * 3]
+                    poses_3d[pose_id][kpt_id * 4] = map_3d[0, int(neck_2d[1]), int(neck_2d[0])] * AVG_PERSON_HEIGHT
+                    poses_3d[pose_id][kpt_id * 4 + 1] = map_3d[1, int(neck_2d[1]), int(neck_2d[0])] * AVG_PERSON_HEIGHT
+                    poses_3d[pose_id][kpt_id * 4 + 2] = map_3d[2, int(neck_2d[1]), int(neck_2d[0])] * AVG_PERSON_HEIGHT
+                    poses_3d[pose_id][kpt_id * 4 + 3] = poses_2d[pose_id][kpt_id * 3 + 2]
+
+                # refine keypoints coordinates at corresponding limbs locations
+                for limb in limbs:
+                    for kpt_id_from in limb:
+                        if poses_2d[pose_id][kpt_id_from * 3 + 2] > keypoint_treshold:
+                            for kpt_id_where in limb:
+                                kpt_from_2d = poses_2d[pose_id][kpt_id_from * 3: kpt_id_from * 3 + 2].astype(int)
+                                map_3d = features[kpt_id_where * 3:(kpt_id_where + 1) * 3]
+                                poses_3d[pose_id][kpt_id_where * 4] =\
```

Comment:
looks like false positive for pylint :) 
```
[W9901(backslash-line-continuation), ] use of \ for line continuation
```
Could you please redesign these statements?